README.md

```diff
@@ -7,3 +7,7 @@ Here's another paragraph, added after I made `master` protected. The
 branch requires pull request reviews before merging, except for
 administrators. Can I push directly?
 
+Hmm. Yes, I can push directly. Is it because I'm an administrator?
+OK, I've changed the configuration so we require pull request reviews
+for administrators too. Can I still push directly?
+
```

Comment:
Hey, this is definitely _the_ place to bikeshed!

Now this gives me a chance to test whether an approval overrides a condemnation.